### ISSUE TITLE
[OSx] Xcode7.3 failing

### ISSUE THREAD

**Issue by RobLoach** [libretro/RetroArch#7539]
https://travis-ci.org/libretro/RetroArch/jobs/451043048

```
CompileXIB OSX/en.lproj/MainMenu.xib
    cd /Users/travis/build/libretro/RetroArch/pkg/apple
    export XCODE_DEVELOPER_USR_PATH=/Applications/Xcode.app/Contents/Developer/usr/bin/..
    /Applications/Xcode.app/Contents/Developer/usr/bin/ibtool --errors --warnings --notices --module RetroArch --output-partial-info-plist /Users/travis/build/libretro/RetroArch/pkg/apple/build/RetroArch.build/Release/RetroArch.build/MainMenu-PartialInfo.plist --auto-activate-custom-fonts --target-device mac --minimum-deployment-target 10.6 --output-format human-readable-text --compile /Users/travis/build/libretro/RetroArch/pkg/apple/build/Release/RetroArch.app/Contents/Resources/en.lproj/MainMenu.nib /Users/travis/build/libretro/RetroArch/pkg/apple/OSX/en.lproj/MainMenu.xib
2018-11-05 20:05:25.762 ibtoold[2367:4949] Exception raised while unarchiving document objects - IBDocumentMissingCapabilityException
/* com.apple.ibtool.errors */
/Users/travis/build/libretro/RetroArch/pkg/apple/OSX/en.lproj/MainMenu.xib: error: The document “(null)” requires Xcode 8.0 or later.
    Recovery Suggestion: This version does not support documents saved in the Xcode 8 format. Open this document with Xcode 8.0 or later.
CpResource /Users/travis/build/libretro/RetroArch/media/retroarch.icns build/Release/RetroArch.app/Contents/Resources/retroarch.icns
    cd /Users/travis/build/libretro/RetroArch/pkg/apple
    builtin-copy -exclude .DS_Store -exclude CVS -exclude .svn -exclude .git -exclude .hg -strip-debug-symbols -strip-tool /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/strip -resolve-src-symlinks /Users/travis/build/libretro/RetroArch/media/retroarch.icns /Users/travis/build/libretro/RetroArch/pkg/apple/build/Release/RetroArch.app/Contents/Resources
** BUILD FAILED **
The following build commands failed:
	CompileXIB OSX/en.lproj/MainMenu.xib
(1 failure)
```

[pkg/apple/OSX/en.lproj/MainMenu.xib](https://github.com/libretro/RetroArch/blob/master/pkg/apple/OSX/en.lproj/MainMenu.xib) needs to be saved in a different format?

Bisect: https://github.com/libretro/RetroArch/pull/7530

**Comment by stuartcarnie** [comment]
@robloach, @twinaphex asked for this change for PPC compatibility. Not sure what he wants to do; perhaps the buildbot needs to be upgraded to Xcode 8.x for that?

**Comment by RobLoach** [comment]
Is there a way we could flip between two different versions based on which Xcode version it's on?

**Comment by CatGreen90** [comment]
The Xcode9.3 build is also silently failing:
https://travis-ci.org/libretro/RetroArch/jobs/450873931
```
** BUILD FAILED **
The following build commands failed:
	CompileC build/RetroArch_Metal.build/Release/RetroArchQt.build/Objects-normal/x86_64/griffin_objc.o /Users/travis/libretro-super/retroarch/griffin/griffin_objc.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)
buildbot job: retroarch: [status: done] [recipes/apple/retroarch-osx-x64]
buildbot message: retroarch: [status: done] [recipes/apple/retroarch-osx-x64]
Packaging
The command "./build-retroarch-metal.sh" exited with 0.
dpl_0
5.02s$ rvm $(travis_internal_ruby) --fuzzy do ruby -S gem install dpl
Warning, new version of rvm available '1.29.4', you are using older version '1.29.3'.
You can disable this warning with:    echo rvm_autoupdate_flag=0 >> ~/.rvmrc
You can enable  auto-update  with:    echo rvm_autoupdate_flag=2 >> ~/.rvmrc
Successfully installed dpl-1.10.4
1 gem installed
56.40s
dpl.1
Installing deploy dependencies
Successfully installed dpl-script-1.10.4
1 gem installed
dpl.2
Preparing deploy
dpl.3
Deploying application
Decrypting SSH key...
Copying binary into dist folder...
/Users/travis/libretro-super/retroarch/pkg/apple/build/Release/RetroArch.app -> ./RetroArch.app
/Users/travis/libretro-super/retroarch/pkg/apple/build/Release/RetroArch.app/Contents -> ./RetroArch.app/Contents
/Users/travis/libretro-super/retroarch/pkg/apple/build/Release/RetroArch.app/Contents/Info.plist -> ./RetroArch.app/Contents/Info.plist
/Users/travis/libretro-super/retroarch/pkg/apple/build/Release/RetroArch.app/Contents/MacOS -> ./RetroArch.app/Contents/MacOS
/Users/travis/libretro-super/retroarch/pkg/apple/build/Release/RetroArch.app/Contents/PkgInfo -> ./RetroArch.app/Contents/PkgInfo
/Users/travis/libretro-super/retroarch/pkg/apple/build/Release/RetroArch.app/Contents/Resources -> ./RetroArch.app/Contents/Resources
/Users/travis/libretro-super/retroarch/pkg/apple/build/Release/RetroArch.app/Contents/Resources/en.lproj -> ./RetroArch.app/Contents/Resources/en.lproj
/Users/travis/libretro-super/retroarch/pkg/apple/build/Release/RetroArch.app/Contents/Resources/en.lproj/InfoPlist.strings -> ./RetroArch.app/Contents/Resources/en.lproj/InfoPlist.strings
/Users/travis/libretro-super/retroarch/pkg/apple/build/Release/RetroArch.app/Contents/Resources/en.lproj/MainMenu_Metal.nib -> ./RetroArch.app/Contents/Resources/en.lproj/MainMenu_Metal.nib
/Users/travis/libretro-super/retroarch/pkg/apple/build/Release/RetroArch.app/Contents/Resources/retroarch.icns -> ./RetroArch.app/Contents/Resources/retroarch.icns
Downloading assets...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
 97  217M   97  211M    0     0  32.4M      0  0:00:06  0:00:06 --:--:-- 33.0M100  217M  100  217M    0     0  32.4M      0  0:00:06  0:00:06 --:--:-- 32.9M
Creating DMG image...
created: /Users/travis/2018-11-05_RetroArch_Metal.dmg
Uploading to server...
Warning: Permanently added '[bot.libretro.com]:12346,[107.182.230.193]:12346' (ECDSA) to the list of known hosts.
building file list ... 
 0 files...1 file to consider
2018-11-05_RetroArch_Metal.dmg
     193.82M  84%   39.66MB/s    0:00:00     230.35M 100%   37.16MB/s    0:00:05 (xfer#1, to-check=0/1)
sent 230.37M bytes  received 42 bytes  35.44M bytes/sec
total size is 230.35M  speedup is 1.00
Done. Your build exited with 0.
```

**Comment by orbea** [comment]
Travis is passing currently so I am going to close this.
